core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -1091,11 +1093,21 @@ private void parseGlobalRole(ConfigurationReader reader, GlobalAuthorizationConf
                permissions = reader.getListAttributeValue(i);
                break;
             }
+            case DESCRIPTION: {
+               description = reader.getAttributeValue(i);
+               break;
+            }
+            case IMPLICIT: {
```

Comment:
Do we need to expose this to the user via the parser? It seems like this is just something we require internally.

Reply:
We need it for the console. The roles are marked not modifiable in those cases

Reply:
Maybe I'm misunderstanding the meaning of a role being marked as "implicit". I understood it to mean that the role is an internally defined role, e.g. those in `DEFAULT_ROLES`, so we expose the "isImplicit" field so that the console is aware of this. Or is there more to this field? I guess my question is, in what use-case would a user define a role with implicit=true?

Reply:
I see what you mean, I can remove the implicit from the xml but still expose it from the rest api